src/Controller/CocktailController.php

```diff
@@ -54,7 +68,26 @@ public function submitName(Request $request): Response
 
         return $this->render('cocktail/submit.html.twig', [
             'cocktailForm' => $form->createView(),
-            'result' => $cocktailName->getCocktailName()
+            'result' => $this->cocktailNameRequest($cocktailName)
         ]);
     }
+
+    private function cocktailNameRequest(CocktailName $cocktailName): string
+    {
+        $response = $this->httpClient->request('GET', self::URL, [
+            'query' => [
+                's' => $cocktailName->getCocktailName()
+            ]
+        ]);
+
+        $parsedResponse = $response->toArray();
+
```

Comment:
Quit from your code early. Don't nests if-elses
```suggestion
if(error case && second error case) {
   return 'err msg';
   }
   
   return 'successful response';
```